src/main/res/values/strings.xml

```diff
@@ -6,12 +6,12 @@
 	<string name="app_host"/>
 
 	<string name="txtAppUrl">CHT-Core URL</string>
```

Comment:
"CHT-Core" isn't meant to be used as the branded tool. We should stick to "CHT server" or "CHT instance" as needed in the context.

```suggestion
	<string name="txtAppUrl">CHT server URL</string>
```